apps/docs/src/guide/types/date-time.md

```diff
@@ -0,0 +1,60 @@
+# Date Time
+
+To allow for easier manipulation of date and time, the SDK exports the `DateTime` class, which is a wrapper around the `Date` class ([reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)). Below we will go over the methods of instantiation, utility functions, and time formats.
+
+Why is this useful?
+Internally the transactions and other time/date assets are encoded using the **TAI64** format. We return a `IDateTime` interface, to allow of easier conversion and formatting between the two formats.
+
+## API
+
+We have a host of static method for **instantiation** of our `DateTime` class.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#create-from-multiple-sources{ts:line-numbers}
+
+### TAI64
+
+`fromTai64` is a _static_ method, that allows the creation of `DateTime` class from a TAI64 string.
+
+`toTai64` is a _instance_ method, that allows the conversion of a `DateTime` class to a `TAI64` string.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#from-tai-64-and-to-tai-64{ts:line-numbers}
+
+### UNIX
+
+`fromUnixMilliseconds` is a _static_ method, that allows the creation of `DateTime` class from a UNIX Milliseconds number.
+
+`toUnixMilliseconds` is a _instance_ method, that allows the conversion of a `DateTime` class to a `UNIX` number in milliseconds.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#from-unix-milliseconds-and-to-unix-milliseconds{ts:line-numbers}
+
+`fromUnixSeconds` is a _static_ method, that allows the creation of `DateTime` class from a UNIX Seconds number.
+
+`toUnixSeconds` is a _instance_ method, that allows the conversion of a `DateTime` class to a `UNIX` number in seconds.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#from-unix-seconds-and-to-unix-seconds{ts:line-numbers}
+
+### Date
+
+The `DateTime` class extends the functionality of the `Date` object (see here), so all method are available for your usages.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#date-object-methods{ts:line-numbers}
+
+## Formats
```

Comment:
For me, that would just be rehashing what has been covered above, and has bloat potential, will leave unless theres a strong sentiment to this change 😄

Reply:
Sorry I mean you should also get rid of what you say above. So rather than splitting up instantiating and formats, just have a subheading for the format and then how to instantiate it.

IMO that would be bless bloated than how it currently is. It would make sense to put everything UNIX related together and everything TAI related together.

Reply:
```md
# DateTime

To allow for easier...

## Instantiating a `DateTime`

We have a host of...

### TAI64

TAI stands for...

Instantiating TAI64...

### UNIX

UNIX intro

Instantiating UNIX...

### Date

Date intro

Instantiating from date...
```

Formats as a sub para seems unnecessary and flows oddly.

Reply:
I personally believe the serperation between the two, gives better visibility to the API, which is what you want surely?

I do agree that the flow isn't ideal, but went with an API first approach with documentation, which I think is more valuable for the user IMO